products/opensuse/main.pm

```diff
@@ -85,6 +85,10 @@ if (check_var('DESKTOP', 'minimalx')) {
     set_var("XDMUSED",     1);
 }
 
+if (check_var('REGRESSION', 'installation')) {
+    set_var('NOAUTOLOGIN', 1);
```

Comment:
Move it to existing 'REGRESSION installation' block?

Reply:
fixed